src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -5417,4 +5417,4 @@ Oluşturma sırasında NuGet'in paketleri indirmesini önlemek için, Visual Stu
     <comment>Do not localize project.json and `PackageReference` 
 0 - project path (project.json file path)</comment>
   </data>
-</root>
\ No newline at end of file
+</root>
```

Comment:
Please reset this file (and it's `.Designer.cs` file) back to the dev branch copy, since there are no "real" changes in it.